PendingReleaseNotes

```diff
@@ -124,6 +124,14 @@
 * RBD: list-watchers C++ API (`Image::list_watchers`) now clears the passed
   `std::list` before potentially appending to it, aligning with the semantics
   of the corresponding C API (`rbd_watchers_list`).
+* Telemetry: Users who are opted-in to telemetry can also opt-in to
+  participating in a leaderboard in the telemetry public
+  dashboards (https://telemetry-public.ceph.com/). Users can now also add a
+  description of the cluster to publicly appear in the leaderboard.
+  For more details, see:
+  https://docs.ceph.com/en/latest/mgr/telemetry/#leaderboard
+  See a sample report with `ceph telemetry preview`.
+  Opt-in with `ceph telemetry on`.
```

Comment:
```suggestion
  Opt-in with `ceph config set mgr mgr/telemetry/leaderboard true`.
```

?

Reply:
@batrick thanks for the suggestion; I clarified the use of the commands